hallo_statlib/conanfile.py

```diff
@@ -41,15 +41,17 @@ def build(self):
         #
         # ***************************************************************************************
 
-        WORK_SPACE = "../workspace"
-        PROJECT_DIR = "../workspace/hello_statlib"
+        WORK_SPACE = "workspace"
+        PROJECT_DIR = "workspace/hello_statlib"
         PROJECT_NAME = "hello_statlib"
-        BUILD_TYPE = "Debug"
+        BUILD_TYPE = str(self.settings.build_type)
         
-        cmd = "eclipse -nosplash -application org.eclipse.cdt.managedbuilder.core.headlessbuild -data " + WORK_SPACE + " -import " + PROJECT_DIR + " " + PROJECT_NAME + "/" + BUILD_TYPE
-        self.run("cmd")
+        cmd = ("eclipse -nosplash -application org.eclipse.cdt.managedbuilder.core.headlessbuild -data "
+               + WORK_SPACE + " -import " + PROJECT_DIR + " -debug")
+        self.run(cmd)
 
     def package(self):
+        # I would recommend "include", "lib"
```

Comment:
Please see the [issue](https://github.com/conan-io/conan/issues/1286) again for I added comments and changed a lot. 

Feel free to fork again like you did here to see the changes you suggest and for me to comment.